### PR TITLE
web page mobile version completed

### PR DESCRIPTION
**Pull request by AdrianScharff**
Add sections "works" and "About" in html
Add all the images required for the project in html
Add the styling as in the template in css
Use grid in the works section and flexbox in each of the sections inside
Use flexbox in the About section

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -16,17 +16,109 @@
...
+            <li><a href="https://www.freecodecamp.org/"> <img src="./images/Githubicon.png" alt=""> </a></li> 
+            <li><a href="https://www.theodinproject.com/"> <img src="./images/Unknownicon.png" alt=""> </a></li> 
          </ul>
     </section>
+
+    <section class="works">
+        <div class="cards" id="cardfirst">
+          <div class="orange"><img src="./images/orange.snapshot.png" alt=""></div>
+            <div class="left-block">
+            <h1>Tonic</h1>
```
Comment (Kossi-stack) [inline index.html]: - [x] Kindly use another heading like h2 ...... hx for subtitles. Use h1 only for the main Title.

File: index.html [hunk 2]
```diff
@@ -16,17 +16,109 @@
...
+                <li><img src="./images/csstag.png" alt=""></li>
+                <li><img src="./images/javascripttag.png" alt=""></li>
+            </ul>
+            <div class="see">See project</div>
+          </div>
+        </div>
+        <div class="cards" id="cardsecond">
+            <div class="beige"><img src="./images/Beige.snapshot.png" alt=""></div>
+            <div class="left-block">
+            <h1>Multi-Post Stories</h1>
```
Comment (Kossi-stack) [inline index.html]: - [x] Kindly use another heading like h2 ...... hx for subtitles. Use h1 only for the main Title.

File: index.html [hunk 3]
```diff
@@ -16,17 +16,109 @@
...
+                <li><img src="./images/csstag.png" alt=""></li>
+                <li><img src="./images/javascripttag.png" alt=""></li>
+            </ul>
+            <div class="see">See project</div>
+            </div>
+        </div>
+        <div class="cards" id="cardthird">
+            <div class="shiny"><img src="./images/shinyorange.snapshot.png" alt=""></div>
+            <div class="left-block">
+            <h1>Tonic</h1>
```
Comment (Kossi-stack) [inline index.html]: - [x] Kindly use another heading like h2 ...... hx for subtitles. Use h1 only for the main Title.

File: index.html [hunk 4]
```diff
@@ -16,17 +16,109 @@
...
+                <li><img src="./images/csstag.png" alt=""></li>
+                <li><img src="./images/javascripttag.png" alt=""></li>
+            </ul>
+            <div class="see">See project</div>
+            </div>
+        </div>
+        <div class="cards" id="cardfourth">
+            <div class="white"><img src="./images/white.snapshot.png" alt=""></div>
+            <div class="left-block">
+            <h1>Multi-Post Stories</h1>
```
Comment (Kossi-stack) [inline index.html]: - [x] Kindly use another heading like h2 ...... hx for subtitles. Use h1 only for the main Title.

File: index.html [hunk 5]
```diff
@@ -16,17 +16,109 @@
...
+                <li><img src="./images/csstag.png" alt=""></li>
+                <li><img src="./images/javascripttag.png" alt=""></li>
+            </ul>
+            <div class="see">See project</div>
+            </div>
+        </div>
+    </section>
+
+    <section class="about">
+      <h1>About Myself</h1>
```
Comment (Kossi-stack) [inline index.html]: - [x] Kindly use another heading like h2 ...... hx for subtitles. Use h1 only for the main Title.

File: index.html [hunk 6]
```diff
@@ -16,17 +16,109 @@
...
+      <p>Hello I’m a software developer! I can help you build a product , feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
+      <a href="#">LET'S CONNECT</a>
+      <ul class="icons2">
+          <li><a href="https://www.freecodecamp.org/"> <img src="./images/Githubicon.png" alt=""> </a></li> 
+          <li><a href="https://www.linkedin.com/in/adri%C3%A1n-ju%C3%A1rez-17a51b263/"> <img src="./images/Linkedinicon.png" alt=""> </a></li>
+              <li><a href="https://www.theodinproject.com/"> <img src="./images/Unknownicon.png" alt=""> </a></li> 
+        <li><a href="https://twitter.com/CryptogodAdrian"> <img src="./images/Twittericon.png" alt=""> </a></li> 
+        <li><a href="https://www.w3schools.com/"> <img src="./images/Mediumicon.png" alt=""> </a></li> 
+     </ul>
+     <div class="get">Get my resume</div>
```
Comment (Kossi-stack) [inline index.html]: - [x] The button (e.g., "Get my resume") is a link `a` or `button` tag,
- Apart from being semantically correct, the `a` tag will let you connect to other sections of the website, using href.

File: index.html [hunk 7]
```diff
@@ -16,17 +16,109 @@
...
+            <li><img src="./images/lancss.png" alt=""></li>
+          </ul>
+       </li>
+        <li class="details"> <span class="frameworks">Frameworks<img class="arrow" src="./images/arrowright.png" alt=""></span> <img src="./images/SeparatorBottom.png"></li>
+        <li class="details"><span class="skills">Skills<img class="arrow" src="./images/arrowright.png" alt=""></span> <img src="./images/SeparatorBottom.png"></li>
+     </ul>
+    </section>
+    <div id="contact">
+      <div id="contacttext">
+        <h1>Contact me</h1>
```
Comment (Kossi-stack) [inline index.html]: - [x] Kindly use another heading like h2 ...... hx for subtitles. Use h1 only for the main Title.

**Review by Kossi-stack — CHANGES_REQUESTED** [review]
## Hi @AdrianScharff,

**Good job so far! 👍🏻**
There are some issues that you still need to work on to go to the next project but you are almost there!💪

68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f556461535559436c723932636a57706556782f67697068792e676966

## To Highlight

✅ Descriptive pull request message
✅ Linter error checks validated
✅ Documented your work professionally
✅ README file is descriptive.
✅ Commit messages are meaningful.

## Required Changes ♻️

 _Check the comments under the review for more changes_.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

_**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by Efiamotu-1 — APPROVED** [review]
Hi @AdrianScharff , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# STATUS: APPROVED 🕺🥳 

giphy


## Highlights :dart: 
* Correct GitHub flow :heavy_check_mark: 
* Good commit messages :heavy_check_mark: 
* All linters passing :heavy_check_mark: 
* Good PR Title & summary :heavy_check_mark: 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

## Important Notes 📝

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

> **Note:** Please, you can now Merge since it is Approved ✅, However if there are any conflicts please resolve them before merging.


## Thank you for your time 🙏 and call me if you need help or clarification ☎️

- This is My **GitHub**: [@Efiamotu-1](https://github.com/Efiamotu-1/)
- This is My **linkedIn**.[Habeeb Efiamotu Musa](https://www.linkedin.com/in/musa-habeeb/)
- This is My **Twitter**: [@EFYAMOTU](https://twitter.com/EFYAMOTU)
- This is My **Slack**: [Habeeb Efiamotu Musa](https://slack.com)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**